utils/themes.ts

```diff
@@ -8,26 +8,30 @@ import {
     DefaultTheme as PaperDefaultTheme,
 } from 'react-native-paper';
 
-const BaseTheme = {
-    roundness: 5,
```

Comment:
why removed roundness?

Reply:
there is a built in roundness, this was overide i think I have added by mistake